core/src/main/scala/org/ensime/core/AnalyzerManager.scala

```diff
@@ -0,0 +1,181 @@
+package org.ensime.core
+
+import scala.concurrent.duration._
+
+import akka.actor._
+import akka.event.LoggingReceive.withLabel
+import org.ensime.api._
+import org.ensime.config.richconfig._
+import org.ensime.util.{ Debouncer, Timing }
+import org.ensime.util.FileUtils.toSourceFileInfo
+import org.ensime.util.file._
+
+final case object SuspendAnalyzer
+
+class AnalyzerManager(
+    broadcaster: ActorRef,
+    analyzerCreator: List[EnsimeProjectId] => Props,
+    implicit val config: EnsimeConfig
+) extends Actor with ActorLogging with Stash {
+
+  // is this okay ?
+  private val suspendAnalyzer = Debouncer.forActor(
```

Comment:
Yes okay, that'll do it.
I just thought, since I was using Debouncer, it might be expensive to have one per analyzer and didn't know if it was wise.